index.js

```diff
@@ -39,7 +45,8 @@
 
     function inSavedCart()
     {
-        return document.getElementsByClassName("cart__heading")[0].textContent != "Pasūtījuma grozs";
+        const yourCartText = ['Your cart', 'Pasūtījuma grozs', 'Корзина'];
+        return !yourCartText.includes(document.getElementsByClassName("cart__heading")[0].textContent);
```

Comment:
there is a lot going on in this line, would be best to split it out to reduce the complexity
```js
const cartHeading = document.getElementsByClassName("cart__heading")[0].textContent;
return !yourCartText.includes(cartHeading);
```